indexer/osm_editor.cpp

```diff
@@ -339,6 +339,26 @@ void Editor::ClearAllLocalEdits()
   Invalidate();
 }
 
+void Editor::OnMapDeregistered(platform::LocalCountryFile const & localFile)
+{
+  // TODO: to add some synchronization mechanism for whole Editor class
+  static mutex mtx;
```

Comment:
Please, move this declaration to the Editor class. We don't need to use the same mutex for all Editor instances, and there's no need in smart static initializer here.